### ISSUE TITLE
Tutorial line sequence issue

### ISSUE THREAD

**Issue by AndrewSmart** [edwardvmills/NURBSlib_EVM#2]
On [Tutorial 0.01 ControlPoly4 and CubicCurve4.md](https://github.com/edwardvmills/NURBSlib_EVM/blob/gh-pages/Tutorial%200.01%20ControlPoly4%20and%20CubicCurve4.md)

In #1 "Draw a sketch with 3 lines connected end to end."  should read something like "Draw a sketch with 3 lines connected end to end in order". If the lines are not created in order it won't work.

**Comment by edwardvmills** [comment]
good catch. i thought the point order was processed internally, but the code just says "error check later". i updated the tutorial phrasing, and will do the actual fix...later (as part of of a larger topology process underway).

Thanks for reporting.
